content/en/blog/_posts/2020-09-03-warnings/index.md

```diff
@@ -328,3 +328,7 @@ A couple areas we're looking at next are warning about [known problematic values
 we cannot reject outright for compatibility reasons, and warning about use of deprecated fields or field values
 (like selectors using beta os/arch node labels, [deprecated in v1.14](/docs/reference/labels-annotations-taints/#beta-kubernetes-io-arch-deprecated)).
 I'm excited to see progress in this area, continuing to make it easier to use Kubernetes.
+
+
+## EDIT
```

Comment:
does this show up in the doc content?

Reply:
As the author, you get to decide (in nearly all cases, anyway).

@Dominik-K: no need to mark this as edited.

Reply:
Ok, fair enough. The edits are transparent in this repo anyway.

Reply:
@liggitt I've removed it.